sonar-plugin-api/pom.xml

```diff
@@ -224,32 +224,37 @@
         </executions>
       </plugin>
       <plugin>
-         <groupId>org.codehaus.gmaven</groupId>
-         <artifactId>groovy-maven-plugin</artifactId>
-         <executions>
-           <execution>
-             <id>compute-version-on-three-fields</id>
-             <phase>generate-resources</phase>
-             <goals>
-               <goal>execute</goal>
-             </goals>
-             <configuration>
-               <source><![CDATA[
+        <groupId>com.github.genthaler</groupId>
+        <artifactId>beanshell-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>compute-3digits-version</id>
```

Comment:
you changed the execution id. The previous one (compute-version-on-three-fields) is more exact (not really digits)